src/Standard.Licensing.Tests/KeyGeneratorTests.cs

```diff
@@ -35,9 +34,9 @@ public class KeyGeneratorTests
         [Test] // See Bug #135
         public void Ensure_To_Not_Generate_Identical_Keys()
         {
-            var passPhrase = "test";
-            var privKeySet = new HashSet<string>();
-            var pubKeySet = new HashSet<string>();
+            string passPhrase = "test";
+            HashSet<string> privKeySet = new HashSet<string>();
+            HashSet<string> pubKeySet = new HashSet<string>();
```

Comment:
Why would you change all of the `var` lines to explicit type declarations? That is not a code cleanup, but a change in the overall standard of how the code is written within this project.